File: Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/utils/CustomTabUtil.java
package com.mentalmachines.droidcon_boston.utils;

import android.content.Context;
import android.net.Uri;
import android.support.customtabs.CustomTabsIntent;
import com.mentalmachines.droidcon_boston.R;

public class CustomTabUtil {

  public static void loadUriInCustomTab(Context context, String uriString) {
    Uri data = Uri.parse(uriString);
    CustomTabsIntent customTabsIntent = new CustomTabsIntent.Builder()
        .setToolbarColor(context.getResources().getColor(R.color.colorPrimary))
        .build();
    customTabsIntent.launchUrl(context, data);
  }
}

File: Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/utils/CustomTabUtil.kt
package com.mentalmachines.droidcon_boston.utils

import android.content.Context
import android.net.Uri
import android.support.customtabs.CustomTabsIntent
import android.support.v4.content.ContextCompat
import com.mentalmachines.droidcon_boston.R

object CustomTabUtil {

    fun loadUriInCustomTab(context: Context, uriString: String) {
        val data = Uri.parse(uriString)
        val customTabsIntent = CustomTabsIntent.Builder()
                .setToolbarColor(ContextCompat.getColor(context, R.color.colorPrimary))
                .build()
        customTabsIntent.launchUrl(context, data)
    }
}
